DrFujiBot_Django/dashboard/admin_commands.py

```diff
@@ -350,9 +350,14 @@ def handle_latestquote(args):
     return output
 
 def handle_addquote(args):
-    quote_text = ' '.join(args)
-    quotee_setting = Setting.objects.filter(key='Quotee')[0]
-    quote_object = Quote(quote_text=quote_text, quotee=quotee_setting.value)
+    if args[0] == "-q":
+        quotee = args[1]
+        quote_text = ' '.join(args[2:])
+        quote_object = Quote(quote_text=quote_text, quotee=quotee)
+    else:
+        quote_text = ' '.join(args)
+        quotee_setting = Setting.objects.filter(key='Quotee')[0]
+        quote_object = Quote(quote_text=quote_text, quotee=quotee_setting.value)
     quote_object.save()
     return 'Quote #' + str(quote_object.id) + ' successfully added'
```

Comment:
This can be written as `return f'Quote # {quote_object.id} successfully added'`.

Reply:
I wasn't aware of this formatting when I wrote the code base, and everywhere else uses string concatenation. I'd rather keep it consistent everywhere for now. Maybe we can go through and change them all at some point.

Reply:
Fair enough, we can do that in a separate pull request